spawn_operator/spawn_operator/lib/spawn_operator/versions/api/v1/actor_host.ex

```diff
@@ -193,6 +193,13 @@ defmodule SpawnOperator.Versions.Api.V1.ActorHost do
                 properties:
                   image:
                     type: string
+                  imagePullSecrets:
```

Comment:
I think a simple string pullSecretsRef would be enough for the operator to know how to build a complete pullSecrets